package.json

```diff
@@ -23,7 +23,13 @@
     "demo:build": "nwb build-react-component src/index.js",
     "lint": "eslint src stories",
     "prepush": "yarn lint",
-    "release": "rm -f .babelrc && yarn build && node scripts/publish-release.js"
+    "release": "rm -f .babelrc && yarn build && node scripts/publish-release.js",
+    "check": "gts check",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
```

Comment:
why not yarn?

Reply:
This was from gts -- I'll clean this up.

Reply:
Yep, that's exactly it. It looks like the upcoming version of gts supports yarn -- I'll probably remove this until it's fully supported.